python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -881,12 +881,12 @@ def send_request():
     not RAY_SERVE_COLLECT_AUTOSCALING_METRICS_ON_HANDLE,
     reason="Only works when collecting request metrics at handle.",
 )
-def test_max_concurrent_queries_set_to_one(serve_instance):
+@pytest.mark.parametrize("use_max_concurrent_queries", [True, False])
```

Comment:
For this test case, we actually need the max ongoing requests to be set to 1 instead of the default, so I can't test neither. I can add a test case for both though.